docs/content/docs/concepts/summarization.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Summarization
+menuPosition: 10
+---
+
+<!-- IMPORTANT: Don't manually edit the auto-generated content below. Instead, edit the source file (look at the path
+     argument) and then run "npm run build:md-magic" from the docs folder. That will update all autogenerated
+     content, so you should commit the resulting changes. -->
+
+<!-- AUTO-GENERATED-CONTENT:START (INCLUDE:path=../../../../packages/runtime/container-runtime/src/summary/README.md) -->
+
+<!-- prettier-ignore-start -->
+<!-- NOTE: This section is automatically generated by embedding the referenced file contents. Do not update these generated contents directly. -->
+
+## Table of contents
+
+-   [Introduction](#introduction)
+    -   [Summary vs snapshot](#summary-vs-snapshot)
+-   [Why do we need summaries?](#why-do-we-need-summaries)
+-   [Who generates summaries?](#who-generates-summaries)
+-   [When are summaries generated?](#when-are-summaries-generated)
+-   [How are summaries generated?](#how-are-summaries-generated)
+    -   [Summary Lifecycle](#summary-lifecycle)
+    -   [Single-commit vs two-commit summaries](#single-commit-vs-two-commit-summaries)
+    -   [Incremental summaries](#incremental-summaries)
+    -   [Resiliency](#resiliency)
+-   [What does a summary look like?](#what-does-a-summary-look-like)
```

Comment:
I don't think you want the TOC - the docs pages already have nav and everything, so this will be redundant. You can add a start argument to the include to omit the top of the readme.